prototype/request_prototype_builder_test.go

```diff
@@ -34,6 +35,7 @@ func TestOnValidRequestConfigRequestPrototypeIsBuilt(t *testing.T) {
 
 	// Then
 	assert.Equal(t, 1, len(requestPrototypes))
+	assert.Equal(t, "test", requestPrototypes[0].Alias)
```

Comment:
Make this 3 fields with 3 different values, just in case.
E.g. "testTo", "aliasTest" etc.